### PR TITLE
Update electroN_P model

### PR DESCRIPTION
**Pull request by luohezhiming**
## Fixes/Resolves:

(replace this with the issue # fixed or resolved, if no issue exists then a brief statement of what this PR does)

## Summary/Motivation:
Update electroN_P model based on latest data, since struvite is not the actual product, we modify the previous reaction model.

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/electrochemical_nutrient_removal/electrochemical_nutrient_removal.py [hunk 1]
```diff
@@ -200,11 +196,11 @@ def add_costing(m):
         / (
             m.fs.costing.utilization_factor
             * pyunits.convert(
-                m.fs.product_struvite.properties[0].flow_mass_comp["struvite"],
+                m.fs.product_salt.properties[0].flow_mass_comp["phosphorus"],
                 to_units=pyunits.kg / m.fs.costing.base_period,
             )
         ),
-        doc="Levelized cost of struvite",
+        doc="Levelized cost of phosphorus recovery",
```
Comment (tarnold17) [inline watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/electrochemical_nutrient_removal/electrochemical_nutrient_removal.py:203]: The `LCOS` variable no longer refers to levelized cost of struvite, so I recommend changing the name of the variable to something more appropriate. Perhaps `LCOP`.

**Review by MarcusHolly — APPROVED** [review]
LGTM - my only comment is that you might want to add the feedback on Box like Tim did for the other project: https://app.box.com/folder/192415445397?s=bvi2vd9nsfhjl7xh5dy0f2gyyyiwq416

File: watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/electrochemical_nutrient_removal/electrochemical_nutrient_removal.py (hunk 1 above)
Comment (luohezhiming) [reply watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/electrochemical_nutrient_removal/electrochemical_nutrient_removal.py:203]: Good suggestion, modified!

**Comment by luohezhiming** [comment]
> LGTM - my only comment is that you might want to add the feedback on Box like Tim did for the other project: https://app.box.com/folder/192415445397?s=bvi2vd9nsfhjl7xh5dy0f2gyyyiwq416

@MarcusHolly Thank you for reminding me, but it seems the linked note you show me only lists differences between WaterTAP3 and WaterTAP. ElectroNP is a brand new model and WaterTAP3 does not have it.

**Comment by MarcusHolly** [comment]
> > LGTM - my only comment is that you might want to add the feedback on Box like Tim did for the other project: https://app.box.com/folder/192415445397?s=bvi2vd9nsfhjl7xh5dy0f2gyyyiwq416
> 
> @MarcusHolly Thank you for reminding me, but it seems the linked note you show me only lists differences between WaterTAP3 and WaterTAP. ElectroNP is a brand new model and WaterTAP3 does not have it.

Not quite sure what you mean by that, but I'd imagine you should still put your feedback in this folder regardless, just to be consistent.

**Comment by luohezhiming** [comment]
> > > LGTM - my only comment is that you might want to add the feedback on Box like Tim did for the other project: https://app.box.com/folder/192415445397?s=bvi2vd9nsfhjl7xh5dy0f2gyyyiwq416
> > 
> > 
> > @MarcusHolly Thank you for reminding me, but it seems the linked note you show me only lists differences between WaterTAP3 and WaterTAP. ElectroNP is a brand new model and WaterTAP3 does not have it.
> 
> Not quite sure what you mean by that, but I'd imagine you should still put your feedback in this folder regardless, just to be consistent.

We previously migrate a lot of old platform WaterTAP3 (https://github.com/NREL/WaterTAP3/blob/master/watertap3/watertap3/WaterTAP3_documentation.pdf) models to WaterTAP, I believe the note is used report difference beween TEA analysis of these models, but AMO is different.

**Comment by tarnold17** [comment]
> LGTM - my only comment is that you might want to add the feedback on Box like Tim did for the other project: https://app.box.com/folder/192415445397?s=bvi2vd9nsfhjl7xh5dy0f2gyyyiwq416

@MarcusHolly, the feedback in this folder is a formal letter that the project team wrote to us with their feedback on the ZO model. We don't have anything nearly as formal for electroN-P, so I don't think there's really anything we can or need to add to Box here.

**Comment by MarcusHolly** [comment]
> > LGTM - my only comment is that you might want to add the feedback on Box like Tim did for the other project: https://app.box.com/folder/192415445397?s=bvi2vd9nsfhjl7xh5dy0f2gyyyiwq416
> 
> @MarcusHolly, the feedback in this folder is a formal letter that the project team wrote to us with their feedback on the ZO model. We don't have anything nearly as formal for electroN-P, so I don't think there's really anything we can or need to add to Box here.

@tarnold17 Oh, alright. I wasn't sure what form the feedback came in for this project, but it seems like there is no need to add anything then. Thanks for clarifying.

**Review by tarnold17 — APPROVED** [review]

**Review by adam-a-a — APPROVED** [review]
LGTM - may want to consider accounting for revenue from sale of phosphorus in a subsequent PR. We may need to update relevant parameter values as well.

**Pull request merged**